spec/migrations/20210519204247_move_miq_set_memberships_spec.rb

```diff
@@ -0,0 +1,77 @@
+require_migration
+
+describe MoveMiqSetMemberships do
+  let(:rel_stub)        { migration_stub(:Relationship) }
+  let(:membership_stub) { migration_stub(:MiqSetMembership) }
+  let(:ns_stub)         { Class.new(ActiveRecord::Base) { self.table_name = "miq_ae_namespaces" } }
+  let(:rp_stub)         { Class.new(ActiveRecord::Base) { self.table_name = "resource_pools" } }
+
+  migration_context :up do
+    context "two sets with one member each" do
+      it 'converts relationship records to set membership table records' do
+        ns1 = ns_stub.create!
+        ns2 = ns_stub.create!
+        ns1_child = ns_stub.create!
+        ns2_child = ns_stub.create!
+        ns1_rel = create_rel(ns1, nil)
+        ns2_rel = create_rel(ns2, nil)
+        create_rel(ns1_child, ancestry_for(ns1_rel))
+        create_rel(ns2_child, ancestry_for(ns2_rel))
+        create_resource_pool
```

Comment:
Why do we have this?  Is this just to prove we _arent_ touching non-membership relationships?

Reply:
Yes, I wasn't fully confident in the SQL I was writing, so I wanted something that was going to be a sanity check to make sure we didn't delete records we shouldn't be.

Reply:
Ok that's good...I think it's a good idea to keep it in.